setup.py

```diff
@@ -80,6 +80,7 @@
 entry_points = {}
 entry_points['console_scripts'] = [
     'gammapy-background-cube = gammapy.scripts.background_cube:main',
+    'gammapy-catalog-query = gammapy.scripts.catalog_query:main',
```

Comment:
There's a merge conflict for this change. Can you please rebase this branch against the latest master version?